sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/Aggregation.java

```diff
@@ -24,43 +28,76 @@
   private static final ThrottlingLogger logger =
       new ThrottlingLogger(Logger.getLogger(Aggregation.class.getName()));
 
-  private static final Aggregation NONE = Aggregation.make("none", unused -> null);
+  private static final Aggregation NONE = new NoAggregation();
```

Comment:
I get a warning from IDEA about possible classloading deadlock by referencing this here.

Reply:
these should be resolved now.   All static initialization uses local classes only, static methods hold the indirect dependencies.